internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -440,7 +440,7 @@ func (k *KafkaManager) updateExpiresAtBasedOnQuotaEntitlement(kafka *dbapi.Kafka
 		}
 
 		// set expires_at to now + grace period days
-		expiresAtTime := time.Now().AddDate(0, 0, billingModel.GracePeriodDays)
+		expiresAtTime := time.Now().AddDate(0, 0, billingModel.GracePeriodDays).Add(time.Hour * 12)
```

Comment:
Just a question for my understanding: How was the 12 hours period choosen? And how is it different to grace period?

Reply:
it was jus an arbitrary number of hours, so kafkas won't be instantly suspended if allowed quota intermittently shows 0

Reply:
Thanks for the clarification. Might be good to extract the "magic" number onto a const/variable with a proper name and comment if possible.

Reply:
makes sense. will do that